paper-textarea.html

```diff
@@ -35,7 +35,7 @@
 <dom-module id="paper-textarea">
   <template>
 
-    <paper-input-container no-label-float$="[[noLabelFloat]]" auto-validate$="[[autoValidate]]" disabled$="[[disabled]]" invalid="[[invalid]]">
+    <paper-input-container no-label-float$="[[noLabelFloat]]" always-float-label="[[_computeAlwaysFloatLabel(alwaysFloatLabel,placeholder)]]" auto-validate$="[[autoValidate]]" disabled$="[[disabled]]" invalid="[[invalid]]">
```

Comment:
Ugh, I wish there was a better way we didn't have to copy this in like 7 elements.

I'm just grumbling.